src/aui/framemanager.cpp

```diff
@@ -1154,14 +1156,30 @@ wxString wxAuiManager::SavePaneInfo(const wxAuiPaneInfo& pane)
     result += wxString::Format(wxT("floatx=%d;"), pane.floating_pos.x);
     result += wxString::Format(wxT("floaty=%d;"), pane.floating_pos.y);
     result += wxString::Format(wxT("floatw=%d;"), pane.floating_size.x);
-    result += wxString::Format(wxT("floath=%d"), pane.floating_size.y);
+    result += wxString::Format(wxT("floath=%d;"), pane.floating_size.y);
+    result += wxString::Format(wxT("floatw_cli=%d;"), pane.floating_client_size.x);
+    result += wxString::Format(wxT("floath_cli=%d"), pane.floating_client_size.y);
 
     return result;
 }
 
 // Load a "pane" with the pane information settings in pane_part
 void wxAuiManager::LoadPaneInfo(wxString pane_part, wxAuiPaneInfo &pane)
 {
+    if (LoadPaneInfoVersioned("layout3", pane_part, pane))
+    {
+        return;
+    }
+    // This is redundant since layout3
+    // recognizes all the layout2 keys,
+    // so should this be removed?
+    LoadPaneInfoVersioned("layout2", pane_part, pane);
```

Comment:
Yes, I think it should be removed, but a comment added above saying that we use "layout3" here because it is capable of loading both "layout2" and "layout3" data and we need to do it in this function for compatibility.